### PR TITLE
M4Singer Recipe for SVS

### PR DESCRIPTION
**Pull request by ftshijt**
This is a new recipe for a new SVS corpus https://m4singer.github.io/

### CODE REVIEW

File: egs/m4singer/voc1/run.sh [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+checkpoint="" # checkpoint path to be used for decoding
+              # if not provided, the latest one will be used
+              # (e.g. <path>/<to>/checkpoint-400000steps.pkl)
+
+# shellcheck disable=SC1091
+. utils/parse_options.sh || exit 1;
+
+train_set=tr_no_dev
+dev_set=dev
+eval_set=eval
```
Comment (kan-bayashi) [inline egs/m4singer/voc1/run.sh:43]:
```suggestion
eval_set="eval"
```

**Comment by ftshijt** [comment]
Hi @kan-bayashi , please pending merge the PR. I will also add the pretrained model link and readme in some later commits. Many thanks for your review!

**Comment by ftshijt** [comment]
Hi @kan-bayashi , I uploaded the pre-trained model and believe it should be good for merging. Please feel free to let me know if it looks good to you.

**Review by kan-bayashi — APPROVED** [review]
Many thanks, @ftshijt !

**Pull request merged**